config/templates/design/src/box.mjs

```diff
@@ -27,33 +14,29 @@ function draftBox({
     .close()
     .attr('class', 'fabric')
 
-  // Complete?
-  if (complete) {
-    points.logo = points.topLeft.shiftFractionTowards(points.bottomRight, 0.5)
-    snippets.logo = new Snippet('logo', points.logo)
-    points.text = points.logo
-      .shift(-90, w / 8)
-      .attr('data-text', 'hello')
-      .attr('data-text-class', 'center')
+  points.logo = points.topLeft.shiftFractionTowards(points.bottomRight, 0.5)
+  snippets.logo = new Snippet('logo', points.logo)
+  points.text = points.logo
+    .shift(-90, w / 8)
+    .attr('data-text', 'hello')
```

Comment:
Since we're improving things, could you change this into:

```js
.addText('hello', 'center')
```

Reply:
Done.